tests/testthat/test-use-lintr.R

```diff
@@ -0,0 +1,24 @@
+context("use_lintr")
+
+test_that("use_lintr('local') creates build-ignored, promised file", {
```

Comment:
Should I add `lintr` to Suggests in DESCRIPTION?

Reply:
No, in general we don't necessarily add a package to Suggests here in usethis, just because usethis offers some support for it. It would not be very sustainable. They way we test related functionality is to use a test mock to get around the presence test. See, for example, `use_tutorial()` is tested vis-a-vis learnr.